web/views/layout.erb

```diff
@@ -29,5 +28,10 @@
     </div>
     <%= erb :_footer %>
     <%= erb :_poll_js %>
+    <script type="text/javascript">
```

Comment:
Please don't use inline scripts, this'd make enforcing Content Security Policies a lot more difficult.